userscripts/odoo-login.js

```diff
@@ -11,7 +11,7 @@
 
 window.addEventListener('load', () => {
   const login = document.querySelector('button[type="submit"]')
-  for (let text of ['admin', 'demo']) {
+  for (let text of ['admin', 'demo', 'login']) {
```

Comment:
It seems that you added the `login` user and not the `portal` user :monocle_face: 
```suggestion
  for (let text of ['admin', 'demo', 'portal']) {
```

Reply:
Sorry, it's fixed now